components/chainhook-sdk/src/observer/mod.rs

```diff
@@ -136,59 +137,6 @@ impl EventObserverConfig {
             _ => unreachable!(),
         }
     }
-
-    pub fn new_using_overrides(
```

Comment:
@lgalabru Just want to verify that this isn't being used by ordhook or some other dep and is safe to delete.

Reply:
Yeah this helper is indeed useful in the context of the sdk. Could you please add a comment?
It's being used in Pyth relayer - convenient way to DRY the code

https://github.com/hirosystems/stacks-pyth-bridge/blob/3afb7a8b174d3cc1a3caeac0890af4379a2e4879/example/relayer/src/config/mod.rs#L79

Reply:
Added back with comment 👍